### ISSUE TITLE
Game crashed to launcher upon save & exit

### ISSUE THREAD

**Issue by CDArena** [Darkosto/SevTech-Ages#1980]
## Issue / Bug
 Game crashed when Exit & Save - game crashed to loader with error message

 https://pastebin.com/wmu6amss

## Expected Behavior
 Regular Save and exit to menu

## Steps to Reproduce (for bugs)

1.
2.

## Logs
* Client/Server Log:
* Crash Log:  https://pastebin.com/wmu6amss

## Client Information

* Modpack Version: 3.06
* Java Version: 1.8.0_171
* Launcher Used: Twitch

* Memory Allocated: 8,175MB (8GB)

* Server/LAN/Single Player: Single player
* Resourcepack Enabled?: default (I checked - the log said something about missing resource pack soarex, but I dont have it installed)
* Optifine Installed?: no

## additional comments: 
Love the game; worth playing despite the crashes
many crashes can be avoided by not doing things too fast, especially when changing dimensions or
completing an age.  startup crashes are pretty frequent for me. I am using the most recent java that
my java installer knows about, and I have sometimes copied the save folder to desktop and
uninstalled/re-installed the mod pack, but they do keep happening. The "crash on save and exit" 
is a new thing for me though; everything did appear to save properly, although I did crash on
restarting to make sure everything saved.

Finally: whats the deal with all those dimensions in the log file? looks like 25 extra dims?

**Comment by justinrusso** [comment]
There is nothing in the log that really indicates anything. If things are working now I'll go ahead and close this out.

Glad to hear you are enjoying the pack! We're going to be releasing an update soon that will hopefully improve performance and likely help with your problems. The dimensions are just loaded, and unloaded, its nothing bad

**Comment by CDArena** [comment]
Just followed the instructions from another post to force the twitch launcher upgrade, just in case. non-intuitively had to swap Back to the Native jar after the upgrade was done though, or I just got Vanilla minecraft. But then the sevtech pack saw a "new launcher". its a bit early to comment definitively, but the first launch I attempted after this did work without crashing (usually it crashes once or twice when I try to start a new session)
